packages/cookie-manager/src/constants.ts

```diff
@@ -33,3 +33,11 @@ export const TRACKER_CATEGORIES: Array<TrackingCategory> = [
   TrackingCategory.PERFORMANCE,
   TrackingCategory.TARGETING,
 ];
+
+export const CATEGORY_EXPIRATION_DAYS: Record<TrackingCategory, number> = {
+  [TrackingCategory.NECESSARY]: 365,
+  [TrackingCategory.FUNCTIONAL]: 365,
+  [TrackingCategory.PERFORMANCE]: 365,
```

Comment:
Please move these to the config.  We will be working on CI checks to apply policies against the config so having it all together would be ideal.